src/status_im2/contexts/chat/photo_selector/album_selector/view.cljs

```diff
@@ -28,10 +28,11 @@
         :number-of-lines 1
         :style           {:margin-right 50}}
        title]
+
       [quo/text
        {:size  :paragraph-2
         :style {:color (colors/theme-colors colors/neutral-50 colors/neutral-40)}}
-       (str count " " (i18n/label :t/images))]]
+       (when count (str count " " (i18n/label :t/images)))]]
```

Comment:
outside the scope of this pr, but shouldn't this just be `(i18n/label :t/images {:count count})` where count is a translation paramater available to the i18n label

Reply:
hm I'm not sure, but yeah probably better